docs/benchmarkdotnet.md

```diff
@@ -114,45 +114,46 @@ And select one of the benchmarks from the list by either entering its number or
 #### Filtering the Benchmarks
 
 You can filter the benchmarks using `--filter $globPattern` console line argument. The filter is **case insensitive**.
-
+<!-- markdownlint-disable MD029 -->
```

Comment:
Is the warning about non-indented bullet points? Can they be fixed instead?

Reply:
no, it's about having a bunch of stuff between the numbers like we do. It's not really fixable to get the effect we want, which is a series of steps with lots of description in them. That's why I chose to disable this warning; it's not really a problem in this context.